<ui version="4.0" stdsetdef="1" >
  <class>AdvancedPage</class>
  <widget class="QWidget" name="advancedPage" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>290</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="3" column="1" >
        <spacer name="spacer2" >
          <property name="sizeHint" >
            <size>
              <width>31</width>
              <height>41</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="timeoutLbl" >
          <property name="text" >
            <string>&amp;Timeout after which a progress dialog appears (in ms):</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>kcfg_Timeout</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="compressionLbl" >
          <property name="text" >
            <string>Default compression &amp;level:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>kcfg_Compression</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="QCheckBox" name="kcfg_UseSshAgent" >
          <property name="text" >
            <string>Utilize a running or start a new ssh-agent process</string>
          </property>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="KIntNumInput" name="kcfg_Compression" >
          <property name="minimum" >
            <number>0</number>
          </property>
          <property name="maximum" >
            <number>9</number>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="KIntNumInput" name="kcfg_Timeout" >
          <property name="minimum" >
            <number>0</number>
          </property>
          <property name="maximum" >
            <number>50000</number>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
    <customwidgets>
    <customwidget>
      <class>KIntNumInput</class>
      <extends>QWidget</extends>
      <header location="global" >knuminput.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >knuminput.h</include>
  </includes>
</ui>
